src/cljs/main/broadfcui/page/external_importer.cljs

```diff
@@ -48,6 +56,96 @@
          you have signed in and completed the user profile registration step you can start using FireCloud."]]]]))
 
 
+(react/defc DockstoreImporter
+  {:render
+   (fn [{:keys [state this]}]
+     (let [{:keys [wdl load-error]} @state]
+       (cond load-error (this :-render-error)
+             wdl (this :-render-export)
+             :else (spinner "Loading WDL..."))))
+   :component-did-mount
+   (fn [{:keys [props state]}]
+     (let [{:keys [id version]} props]
+       (ajax/call {:url (str "https://staging.dockstore.org:8443/api/ga4gh/v1/tools/"
+                             (js/encodeURIComponent (str "#workflow/" id))
+                             "/versions/"
+                             (js/encodeURIComponent version)
+                             "/WDL/descriptor")
+                   :on-done (fn [{:keys [success? get-parsed-response]}]
+                              (if success?
+                                (swap! state assoc :wdl (:descriptor (get-parsed-response)))
+                                (swap! state assoc :load-error (get-parsed-response false))))})))
+   :-render-error
+   (fn [{:keys [state]}]
+     [:div {}
+      [:div {:style {:margin "0.5rem 0"}} "Error loading WDL:"]
+      [comps/ErrorViewer {:error (:load-error @state)}]])
+   :-render-export
+   (fn [{:keys [props state locals this]}]
+     (let [{:keys [id]} props
+           {:keys [banner server-error wdl]} @state]
+       [:div {}
+        (blocker banner)
+        [:div {:style {:marginBottom "1rem"}}
+         (icons/render-icon {:style {:marginRight "0.5rem" :color (:state-warning style/colors)}}
+           :warning)
+         "Please note: Dockstore cannot guarantee that WDL and Docker that is referenced by this Workflow will not change. We advise you to review the WDL before future runs."]
```

Comment:
grammar: "cannot guarantee that the WDL and Docker image referenced by this Workflow"